.github/workflows/deploy-libre-testnet.yaml

```diff
@@ -83,6 +83,8 @@ jobs:
           HAPI_EOS_API_NETWORK_NAME: libre
           HAPI_EOS_API_ENDPOINTS: '["https://libre-testnet.edenia.cloud","https://api.testnet.libre.cryptobloks.io","https://libre-testnet.eosphere.io"]'
           HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT: ''
```

Comment:
do we need a value for the SHIP endpoint?

Reply:
@Torresmorah  please add `ws://api-node.libre-testnet:8080` as value for `HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT`